README.md

```diff
@@ -76,14 +76,65 @@ This code is still research-quality.  It is not (yet) suitable for
 deployment.  The development roadmap can be found in the
 [Milestones][gh_milestones] section of the [Github repo][gh_repo].
 
-## Tests
+## Example
 
-Run tests with `cargo test`.
+```
+# extern crate rand;
```

Comment:
Is this snippet in README really a doctest? if it's not, we can remove all pound-prefixed lines entirely.

Reply:
Yes, it is: the README is included as a doc-str for the crate's root module, and this snippet is run as a doctest in that context.

Reply:
it is a doctest! it runs with `cargo test --doc`

Reply:
Cool!